src/tooling/piral-cli/src/injectors/pilet.ts

```diff
@@ -65,14 +66,14 @@ export default class PiletInjector implements KrasInjector {
   setOptions() {}
 
   getMetaOf(index: number) {
-    const { api, pilets } = this.config;
+    const { port, api, pilets } = this.config;
     const { bundler, root, requireRef } = pilets[index];
     const def = JSON.parse(readFileSync(join(root, 'package.json'), 'utf8'));
     const file = bundler.bundle.name.replace(/^\//, '');
     return {
       name: def.name,
       version: def.version,
-      link: `${api}/${index}/${file}`,
+      link: `http://localhost:${port}${api}/${index}/${file}`,
```

Comment:
Hi, here is the [background](https://github.com/FlorianRappl/piral-webpack-tools/pull/18)